front/lib/api/assistant/configuration.ts

```diff
@@ -753,7 +753,7 @@ async function isSelfHostedImageWithValidContentType(pictureUrl: string) {
 
 type AgentConfigurationWithoutActionsType = Omit<
   AgentConfigurationType,
-  "actions"
```

Comment:
I'm removing the `generation` too from this part since we now generate the generation after the configuration. I decided not to rename the class since we will soon consider generation as an action. OK for you?